src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,33 @@
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.exception.DukeInvalidDateException;
+import duke.exception.DukeInvalidTaskException;
+import duke.tasks.Deadline;
+
+
+public class DeadlineTest {
+    @Test
+    public void toString_correctDateFormat_properDateFormat() {
+        try {
+            Deadline task = new Deadline("hi", "2020-01-01");
+            String expected = "[D][✗] hi (by: Jan 1 2020)";
+            assertEquals(expected, task.toString());
+        } catch (DukeInvalidTaskException | DukeInvalidDateException e) {
```

Comment:
If an exception is caught and you weren't expecting it, will this cause the test to pass mistakenly? You might want to check that out.